pkg/reconciler/revision/resources/queue.go

```diff
@@ -355,7 +355,12 @@ func applyReadinessProbeDefaults(p *corev1.Probe, port int32) {
 		p.TCPSocket.Host = localAddress
 		p.TCPSocket.Port = intstr.FromInt(int(port))
 	case p.Exec != nil:
-		//User-defined ExecProbe will still be run on user-container.
+		// User-defined ExecProbe will still be run on user-container.
+		// Translate to TCP probes.
```

Comment:
I think we initially had something like this on #4731 but it was decided to leave it out since it didn't necessarily tell us what we needed to know. See: https://github.com/knative/serving/pull/4731#discussion_r303414705